datastore/unit_tests/test_entity.py

```diff
@@ -190,10 +190,13 @@ def test___repr___no_key_empty(self):
 
     def test___repr___w_key_non_empty(self):
         key = _Key()
-        key._path = '/bar/baz'
+        flat_path = ('bar', 12, 'baz', 'himom')
+        key._flat_path = flat_path
         entity = self._make_one(key=key)
-        entity['foo'] = 'Foo'
-        self.assertEqual(repr(entity), "<Entity/bar/baz {'foo': 'Foo'}>")
+        entity_vals = {'foo': 'Foo'}
+        entity.update(entity_vals)
+        expected = '<Entity%s %s>' % (flat_path, entity_vals)
```

Comment:
I don't see that the tuple form is any improvement in "comprehensibility" over the path form, at least for "human" consumption.